HW3.R

```diff
@@ -0,0 +1,108 @@
+#################
+######HW 3#######
+#################
+
+######CLASS STUB#########
+mysqrt <- function (x=64, tol=.0001, myguess=10)
+{
+newguess <- myguess
+error <- tol + 1
+
+while (error> tol) {
+approxy<- x/newguess
+newguess <- (newguess+approxy)/2
+error = newguess - approxy
+print(error < tol)
+print(newguess)
+print(error)
+}
+}
+
+mysqrt()
+
+###########STUB##########
+mysqrt <- function( x=64, tol=.00001, myguess=10){
+
+	newguess <- myguess
+	error <- tol + 1
+
+	while ( error > tol ) { 
+	  approxy <- x/newguess
+	  newguess <- (newguess+approxy)/2 
+	  error = newguess - approxy
+	  print(error < tol) 
+	  print(newguess)
+	  print(error)
+	}
+	return( newguess )
+
+}
```

Comment:
For a final submission, this should not be here. Bad form to have several functions with the same name! Pick one, clean it up. @izzygaw